engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -516,14 +536,139 @@ public String spawnBlock(@Sender EntityRef sender, @CommandParam("blockName") St
+        }
+
+        // this returns the block you have spawned and the amount
+        return "Dropped " + value + " Blocks :)";
+    }
+
+    @Command(shortDescription = "Spawns 10 bowling pin in a regular patter (1,2,3,4)", helpText = "just enter the command prepare and your block name",
+            runOnServer = true, requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String prepare  (@Sender EntityRef sender, @CommandParam("blockName") String blockName) {
+
+        //loop for getting 10 blocks
+        for (int m = 0 ;m< 10; m++) {
+
+
+            //SpawnPrefabActionComponent spawnInfo = sender.getComponent(SpawnPrefabActionComponent.class);
+
+
+            ClientComponent clientComponent = sender.getComponent(ClientComponent.class);
+            LocationComponent characterLocation = clientComponent.character.getComponent(LocationComponent.class);
+
+            Vector3f spawnPos = characterLocation.getWorldPosition();
+            Vector3f offset = characterLocation.getWorldDirection();
+
+
+            offset.scale(0);
+            // a is the distance betweeen the pins and coordinate difference in z vector
+            //xyz are the normal coordinates
+
+            float y = 6.0f; float x = 2.0f; float z =  2.0f; float a = 0.3f; float b = 0.5f;
+            // there is no guarantee that the pins will stand straight, but the range for ir to stand up right is from the vector y between (5.8 - 6.1)
+            float x1 = 0; float x2 = 0; float x3 = 0; float x4 = 0; float x5 = 0; float x6 = 0; float x7 = 0; float x8 = 0; float x9 = 0; float x10 = 0; float z1 = 0; float z2 = 0; float z3 = 0; float z4 = 0; float z5 = 0; float z6 = 0; float z7 = 0; float z8 = 0; float z9 = 0; float z10 = 0;
+
+
+            x1 = x; z1 = z;
+            x2 = x + a; z2 = z+b;
+            x3 = x-a; z3 = z2;
+            x4 = x1+a; z4 = z3+b;
+            x5 = x1; z5 = z4;
+            x6 = x3-a; z6 = z5;
+            x7 = x4+a; z7 = z6+b;
+            x8 = x2; z8 = z7;
+            x9 = x3; z9 = z8;
+            x10 = x6-a; z10 = z8;
 
+            //I know I could have used if else but eh..
+            if (m == 0) {
+                spawnPos.add(x1, y, z1);
+            }
+            if (m == 1) {
+                spawnPos.add(x2, y, z2);
+            }
+
+            if (m == 2) {
+                spawnPos.add(x3, y, z3);
+            }
+            if (m == 3) {
+                spawnPos.add(x4, y, z4);
+            }
+            if (m == 4) {
+                spawnPos.add(x5, y, z5);
+            }
+            if (m == 5) {
+                spawnPos.add(x6, y, z6);
+            }
+            if (m == 6) {
+                spawnPos.add(x7, y, z7);
+            }
+            if (m == 7) {
+                spawnPos.add(x8, y, z8);
+            }
+            if (m == 8) {
+                spawnPos.add(x9, y, z9);
+            }
+            if (m == 9) {
+                spawnPos.add(x10, y, z10);
+            }
```

Comment:
Please use a switch-case here instead of chaining if statements.

However this is pointing to you using loops incorrectly. This whole section could be simplified drastically